apis/sgv2-graphqlapi/src/test/java/io/stargate/sgv2/graphql/integration/util/GraphqlIntegrationTest.java

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright The Stargate Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+package io.stargate.sgv2.graphql.integration.util;
+
+import io.quarkus.test.common.http.TestHTTPResource;
+import io.stargate.sgv2.common.CqlEnabledIntegrationTestBase;
+
+public abstract class GraphqlIntegrationTest extends CqlEnabledIntegrationTestBase {
+
+  @TestHTTPResource protected String baseUrl;
```

Comment:
will `@TestHTTPResource` work with `@QuarkusIntegrationTest`?

Reply:
Yes, I've confirmed that it works.